Manifest/SME/k.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Why is this `k.json`? Was this tested?

Reply:
Yes it was tested. Might be missed while pushing changes from ADO